packages/near-sdk-js/src/collections/vector.ts

```diff
@@ -36,6 +47,47 @@ export class Vector<DataType> {
     return this.length === 0;
   }
 
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  /* eslint-disable @typescript-eslint/no-empty-function */
+  subtype(): any {}
+
+  set_reconstructor(
```

Comment:
Can `Vector` inherit from `SubType` as well to avoid duplicate this method?

Reply:
ok, previous version I meet `Circular dependency` error in compiling code,but currently it compiles successfully.